templates/collection_view_screen/app/stylesheets/name_cell_stylesheet.rb

```diff
@@ -0,0 +1,13 @@
+module <%= @name_camel_case %>CellStylesheet
+  def cell_size
+    {w: 96, h: 96}
+  end
+
+  def <%= @name %>_cell(st)
+    st.frame = cell_size
+    st.background_color = color.random
```

Comment:
do we really want to set these values in a template?  I know that users are going to want something here?  it works well if thats the values you want, but I generally prefer to default values that are going to stay instead of making something a user should need to remove every use of the generator.